tools/buildtools/ambertools/tleap.xml

```diff
@@ -0,0 +1,1987 @@
+          </repeat>
+          <repeat name="tleap_cmds">
+            <conditional name="tleap_cond">
+                 <param name="tleap_cmd" value="loadAmberParams" />
+                 <conditional name="file_source" >
+                   <param name="file_source_selector" value="history"/>
+                   <param name="arg_filename" value="ZAFF.frcmod"/>
+                 </conditional>
+            </conditional>
+          </repeat>
+          <repeat name="tleap_cmds">
+            <conditional name="tleap_cond">
+                 <param name="tleap_cmd" value="loadPdb" />
+                   <param name="loadPdb_assign" value="mol" />
+                   <param name="arg_filename" value="sarscov2_helicase_ZincBindingDomain.pdb"/>
+            </conditional>
+          </repeat>
+          <repeat name="tleap_cmds">
+            <conditional name="tleap_cond">
+                 <param name="tleap_cmd" value="bond" />
+                   <param name="arg_atom1" value="mol.114.ZN" />
+                   <param name="arg_atom2" value="mol.5.SG" /> 
+            </conditional>
+          </repeat>
+          <repeat name="tleap_cmds">
+            <conditional name="tleap_cond">
+                 <param name="tleap_cmd" value="set" />    
+                   <param name="arg_variable" value="mol" />
+                   <param name="arg_container" value="box" />
+                   <param name="arg_dim" value="12" />
+            </conditional>
+          </repeat>
+          <repeat name="tleap_cmds">
+            <conditional name="tleap_cond">
+                 <param name="tleap_cmd" value="addIons" />
+                   <param name="arg_variable" value="mol" />
+                   <param name="arg_ion1" value="Cl-" />
+                   <param name="arg__ion1" value="0" />
+                   <param name="arg_ion2" value="Na+" />
+                   <param name="arg__ion2" value="0" />
+            </conditional>
+          </repeat>
+          <repeat name="tleap_cmds">
+            <conditional name="tleap_cond">
+                 <param name="tleap_cmd" value="saveAmberParm" />
+                   <param name="arg_unit" value="mol" />
+            </conditional>
+          </repeat>
+          <repeat name="tleap_cmds">
+            <conditional name="tleap_cond">
+                 <param name="tleap_cmd" value="savePdb" />
+                   <param name="arg_unit" value="mol" />
+            </conditional>
+          </repeat>
+               <output name="output_tleap_in" >
+                    <assert_contents>
+                         <has_text text="bond mol.114.ZN mol.5.SG" />
+                    </assert_contents>
+               </output>
+               <output name="output_tleap" >
+                    <assert_contents>
+                         <has_text text="Welcome to LEaP!" />
+                    </assert_contents>
+               </output>
+               <output_collection name="output_top" type="list">
```

Comment:
you can also test for the number of elements in this collection, I think with "count"

Reply:
do you have a link to an example on how to use "count" when testing a collection? thanks man.

Reply:
`<output_collection name="output_top" type="list" count="3">` should work, here are some examples from IUC: https://github.com/galaxyproject/tools-iuc/search?l=XML&q=output_collection+count

Reply:
Ok, thanks @simonbray!

Reply:
added count lines